### DISCUSSION TITLE
How to renew etcd certificates in okd 4?

### DISCUSSION THREAD

**Discussion by rakeshk121** [okd-project/okd#775] [Q&A]
Hello,
Im checking on how to renew etcd certificates, from the official documentation I don't see any information regarding renewal https://docs.okd.io/latest/security/certificate_types_descriptions/etcd-certificates.html

Please help me by providing info how to renew the etcd certificates. Thanks in Advance.

**Comment by duritong** [comment]
This happens automatically through the operator: https://docs.okd.io/latest/security/certificate_types_descriptions/etcd-certificates.html#management

**Reply by rakeshk121** [reply]
Thanks for the info
